coderd/workspaceagents.go

```diff
@@ -412,6 +444,9 @@ func (api *API) workspaceAgentStartupLogs(rw http.ResponseWriter, r *http.Reques
 			return
 		}
 	}
+	if after < 0 {
```

Comment:
Under which circumstances `after` is < 0?

Reply:
Perhaps this should return an error, test cases were written with `-500`, and I didn't know the reason. I can remove this check as it doesn't matter. Added it while trying to figure out the memory bug in the sdk.

Reply:
Fixed in [`a1a40f3` (#8029)](https://github.com/coder/coder/pull/8029/commits/a1a40f37f72ee37ddb808ed7024815ad099b4cd0)